public/wp-content/themes/newsup-child/functions.php

```diff
@@ -50,6 +50,82 @@ function set_custom_roles(){
             'read' => true,
             'read_private_pages' => true,
             'read_private_posts' => true,
-            'upload_files' => true    
+            'upload_files' => true
         ) );
-    }
\ No newline at end of file
+    }
+
+// These functions below override existing hooks in
+// the parent theme. These functions specifically
+// deal with injected HTML/PHP code for the use of
+// content sections such as the Homepage Banner, and
+// other advanced content blocks.
+//
+// NOTE:  CHILD THEMES LOAD BEFORE PARENT THEMES AND
+//        THUS CAN OVERRIDE THEIR FUNCTIONS. THIS IS
+//        CALLED REPLUGGING A HOOK FROM THE PARENT.
+//
+// This function is called whenever the theme requests
+// the content section for the banner of trending posts
+// on the homepage. This function is originall located
+// in ../newsup/inc/hooks/hooks.php and has been copied
+// here to be overriden.
+if (!function_exists('newsup_banner_trending_posts')):
+    /**
+     *
+     * @since newsup 1.0.0
+     *
+     */
+    function newsup_banner_exclusive_posts()  {
+            if (is_front_page() || is_home()) {
+                $show_flash_news_section = newsup_get_option('show_flash_news_section');
+            if ($show_flash_news_section):
+        ?>
+            <section class="mg-latest-news-sec">
+                <?php
+                $category = newsup_get_option('select_flash_news_category');
+                $number_of_posts = newsup_get_option('number_of_flash_news');
+                $newsup_ticker_news_title = newsup_get_option('flash_news_title');
+
+                $all_posts = newsup_get_posts($number_of_posts, $category);
+                $show_trending = true;
+                $count = 1;
+
+                // This code will provide a query on Custom Post Type: Anime
+                // to be used by this module. Modifications were made to use
+                // the custom query instead of the original $all_posts obejct.
+                $anime_posts = new WP_Query(array('post_type' => 'anime', 'orderby' => 'date'));
```

Comment:
This is the new query, very simple as we learned in class, it returns all Anime posts, ordered by date added to the website. I hope to change this to order by date it was released using "anime_release_date", a custom field I made using ACF.